sn_node/src/replication.rs

```diff
@@ -55,7 +55,14 @@ impl Node {
             None => return,
         };
         trace!("Being out of closest range, replicating {addr:?} to {close_peers:?}");
+        let mut peers_updated = 0;
         for peer in close_peers.iter() {
+            peers_updated += 1;
```

Comment:
I think the root reason is that libp2p's `get_closest_local_peers` function is little bit misleading that it actually get ALL peers just sort by distance. 
The name is very misleading as for us closest only means `CLOSE_GROUP_SIZE` peers.

To fix it more properly, shall we do a sized chop at the place of calling libp2p's `get_closest_local_peers`
i.e. at the place of handling `SwarmCmd::GetClosestLocalPeers`